process-controller/src/main/java/org/jboss/as/process/ProcessControllerClient.java

```diff
@@ -110,11 +110,11 @@ public void handleMessage(final Connection connection, final InputStream dataStr
                         final Map<String, ProcessInfo> inventory = new HashMap<String, ProcessInfo>();
                         for (int i = 0; i < cnt; i++) {
                             final String processName = readUTFZBytes(dataStream);
-                            final byte[] processAuthCode = new byte[16];
-                            readFully(dataStream, processAuthCode);
+                            final byte[] processAuthBytes = new byte[ProcessController.AUTH_BYTES_ENCODED_LENGTH];
+                            readFully(dataStream, processAuthBytes);
                             final boolean processRunning = StreamUtils.readBoolean(dataStream);
                             final boolean processStopping = StreamUtils.readBoolean(dataStream);
-                            inventory.put(processName, new ProcessInfo(processName, processAuthCode, processRunning, processStopping));
+                            inventory.put(processName, new ProcessInfo(processName, new String(processAuthBytes), processRunning, processStopping));
```

Comment:
I believe IDEA has an inspection for this that you can enable.  Also for locales too, which can cause similar issues.